backend/src/index.ts

```diff
@@ -0,0 +1,50 @@
+import dotenv from "dotenv";
+import express, { Request, Response } from "express";
+import cors from "cors";
+import helmet from "helmet";
+import http from "http";
+import { Server as IOServer, Socket } from "socket.io";
+
+dotenv.config();
+
+// Setup Environment Variables
+if (!process.env.PORT) {
```

Comment:
Can you add some instructions somewhere, probably in the root README, for setting up the environment variables?

Reply:
I've changed the way the config is handled to use [convict](https://github.com/mozilla/node-convict/tree/master/packages/convict). It allows defaults now & .env is only to override those but let me know if you still feel a README is useful 👌🏼